pyproject.toml

```diff
@@ -2,4 +2,7 @@
 requires = ["setuptools", "setuptools_scm", "wheel"]
 
 [tool.pytest.ini_options]
-addopts = "-k 'not spe1'"
+addopts = "-m 'not spe1'"
+markers = [
+    "spe1: marks tests needing results from ert forward model spe1",
```

Comment:
to be pedantic, maybe rephrase to "from ert model of the spe1 test case". 

(an ert model consists (usually) of multiple forward models, and spe1 is really an external thing from Society of Petroleum Engineers)